messages/md.deploy.json

```diff
@@ -0,0 +1,47 @@
+    "validate a deployment so the ID can be used for a quick deploy",
+    "  sfdx force:mdapi:beta:deploy -d some/path -w 1000 -c --testlevel RunAllTestsInOrg",
+    "quick deploy using a previously validated deployment",
+    "  sfdx force:mdapi:beta:deploy -q MyValidatedId"
+  ],
+  "flags": {
+    "checkOnly": "validate deploy but don’t save to the org",
+    "deployDir": "root of directory tree of files to deploy",
+    "wait": "wait time for command to finish in minutes (default: %s)",
+    "jobId": "job ID of the deployment to check; required if you’ve never deployed using Salesforce CLI; if not specified, defaults to your most recent CLI deployment",
+    "testLevel": "deployment testing level",
+    "runTests": "tests to run if --testlevel RunSpecifiedTests",
+    "ignoreErrors": "ignore any errors and do not roll back deployment",
+    "ignoreWarnings": "whether a warning will allow a deployment to complete successfully",
+    "zipFile": "path to .zip file of metadata to deploy",
+    "verbose": "verbose output of deploy results",
+    "validatedDeployRequestId": "request ID of the validated deployment to run a Quick Deploy",
+    "singlePackage": "Indicates that the zip file points to a directory structure for a single package",
+    "soapDeploy": "deploy metadata with SOAP API instead of REST API"
+  },
+  "flagsLong": {
+    "checkOnly": "Validates the deployed metadata and runs all Apex tests, but prevents the deployment from being saved to the org.\nIf you change a field type from Master-Detail to Lookup or vice versa, that change isn’t supported when using the --checkonly parameter to test a deployment (validation). This kind of change isn’t supported for test deployments to avoid the risk of data loss or corruption. If a change that isn’t supported for test deployments is included in a deployment package, the test deployment fails and issues an error.\nIf your deployment package changes a field type from Master-Detail to Lookup or vice versa, you can still validate the changes prior to deploying to Production by performing a full deployment to another test Sandbox. A full deployment includes a validation of the changes as part of the deployment process.\nNote: A Metadata API deployment that includes Master-Detail relationships deletes all detail records in the Recycle Bin in the following cases.\n1. For a deployment with a new Master-Detail field, soft delete (send to the Recycle Bin) all detail records before proceeding to deploy the Master-Detail field, or the deployment fails. During the deployment, detail records are permanently deleted from the Recycle Bin and cannot be recovered.\n2. For a deployment that converts a Lookup field relationship to a Master-Detail relationship, detail records must reference a master record or be soft-deleted (sent to the Recycle Bin) for the deployment to succeed. However, a successful deployment permanently deletes any detail records in the Recycle Bin.",
```

Comment:
```suggestion
    "checkOnly": "IMPORTANT: Where possible, we changed noninclusive terms to align with our company value of Equality. We maintained certain terms to avoid any effect on customer implementations.\n\nValidates the deployed metadata and runs all Apex tests, but prevents the deployment from being saved to the org.\nIf you change a field type from Master-Detail to Lookup or vice versa, that change isn’t supported when using the --checkonly parameter to test a deployment (validation). This kind of change isn’t supported for test deployments to avoid the risk of data loss or corruption. If a change that isn’t supported for test deployments is included in a deployment package, the test deployment fails and issues an error.\nIf your deployment package changes a field type from Master-Detail to Lookup or vice versa, you can still validate the changes prior to deploying to Production by performing a full deployment to another test Sandbox. A full deployment includes a validation of the changes as part of the deployment process.\nNote: A Metadata API deployment that includes Master-Detail relationships deletes all detail records in the Recycle Bin in the following cases.\n1. For a deployment with a new Master-Detail field, soft delete (send to the Recycle Bin) all detail records before proceeding to deploy the Master-Detail field, or the deployment fails. During the deployment, detail records are permanently deleted from the Recycle Bin and cannot be recovered.\n2. For a deployment that converts a Lookup field relationship to a Master-Detail relationship, detail records must reference a master record or be soft-deleted (sent to the Recycle Bin) for the deployment to succeed. However, a successful deployment permanently deletes any detail records in the Recycle Bin.",
```